apps/src/shareWarnings.js

```diff
@@ -52,7 +52,7 @@ function handleShareWarningsTooYoung(onTooYoung) {
   utils.trySetLocalStorage('is13Plus', 'false');
   if (onTooYoung) {
     onTooYoung();
-  } else {
+  } else if (!IN_UNIT_TEST) {
```

Comment:
Not sure I love having "IN_UNIT_TEST" checks in our production code, though I'm also not sure I see an immediately better answer.